<H2>Description</H2>

<div class="warn-note" style="border: 2px solid #9e5900; padding: 5px 20px; background-color: #ffe9cc;">
 <P><span style="font-weight: bold; color: #c70000;">NOTE:</span><br> 
 Decipher is not updating hg19 anymore. This data is outdated. New Decipher data will not appear on hg19. Please use the hg38 track !
 </P>
 <P>To go to the corresponding location in hg38 now, go back to the chromosome
 view, click in the menu bar "View &gt; In other Genomes" and select the hg38
 assembly. </p>

 <p><b>Data Display Agreement Notice</b><br>
 These data are only available for display in the Browser, and not for bulk
 download.  Access to bulk data may be obtained directly from DECIPHER
 (<a href='https://www.deciphergenomics.org/about/data-sharing' target='_blank'
 >https://www.deciphergenomics.org/about/data-sharing</a>) and is subject to a
 Data Access Agreement, in which the user certifies that no attempt to
 identify individual patients will be undertaken.  The same restrictions
 apply to the public data displayed at UCSC in the UCSC Genome Browser;
 no one is authorized to attempt to identify patients by any means.
 </p>
 <p>These data are made available as soon as possible and may be a
 pre-publication release.  For information on the proper use of DECIPHER
 data, please see <a href='https://www.deciphergenomics.org/about/data-sharing'
 target='_blank'>https://www.deciphergenomics.org/about/data-sharing</a>.
 </p>
 <p>The DECIPHER consortium provides these data in good faith as a research
 tool, but without verifying the accuracy, clinical validity, or utility of
 the data.  The DECIPHER consortium makes no warranty, express or implied,
 nor assumes any legal liability or responsibility for any purpose for
 which the data are used.
 </p>
</div>

<P>
The 
<A HREF="https://decipher.sanger.ac.uk" TARGET=_BLANK>DECIPHER</A>
database of submicroscopic chromosomal imbalance 
collects clinical information about chromosomal 
microdeletions/duplications/insertions, translocations and inversions, 
and displays this information on the human genome map.
<p>
This track shows genomic regions of reported cases and their 
associated phenotype information.  All data have passed the strict
consent requirements of the DECIPHER project and are approved for
unrestricted public release.  Clicking the Patient View ID link
brings up a more detailed informational page on the patient at the 
DECIPHER web site. 

<H2>Display Conventions and Configuration</H2>
<P>
The genomic locations of DECIPHER variants are labeled with the DECIPHER variant descriptions. 
<b>Mouseover</b> on items shows variant details, clinical interpretation, and associated conditions. 
Further information on each variant is displayed on the details page by a click onto any variant. 
</p>

<P>
For the <b>CNVs track</b>, the entries are colored by the <b>type of variant</b>:
<ul>
 <li><b><font color="red">red</font></b> for loss</li>
 <li><b><font color="blue">blue</font></b> for gain</li>
 <li><b><font color="grey">grey</font></b> for amplification</li>
</ul>
</P>

<P>
A light-to-dark color gradient indicates the <b>clinical significance</b> of each variant, with 
the lightest shade being benign, to the darkest shade being pathogenic. Detailed information on the 
CNV color code is described <a href="../../goldenPath/help/hgCnvColoring.html">here</a>.
Items can be filtered according to the size of the variant, variant type, and clinical significance 
using the track <b>Configure</b> options.
</P>
 
<P>
For the <b>SNVs track</b>, the entries are colored according to the estimated <b>clinical significance</b> 
of the variant:
<ul>
 <li><b><font color="black">black</font></b> for likely or definitely pathogenic</li>
 <li><b><font color="#888">dark grey</font></b> for uncertain or unknown</li>
 <li><b><font color="#c8c8c8">light grey</font></b> for likely or definitely benign</li>
 </ul>
</P>

<H2>Method</H2>
<P>
Data provided by the DECIPHER project group are imported and processed
to create a simple BED track to annotate the genomic regions associated
with individual patients.
</P>


<H2>Contact</H2>
<P>
For more information on DECIPHER, please contact
<A HREF="mailto:&#99;&#111;n&#116;&#97;c&#116;&#64;&#100;&#101;&#99;&#105;p&#104;&#101;&#114;&#103;&#101;&#110;&#111;&#109;&#105;c&#115;.
&#111;&#114;g">
&#99;&#111;n&#116;&#97;c&#116;&#64;&#100;&#101;&#99;&#105;p&#104;&#101;&#114;&#103;&#101;&#110;&#111;&#109;&#105;c&#115;.
&#111;&#114;g</A>
</P>

<H2>References</H2>
<p>
Firth HV, Richards SM, Bevan AP, Clayton S, Corpas M, Rajan D, Van Vooren S, Moreau Y, Pettett RM,
Carter NP.
<a href="https://www.cell.com/ajhg/abstract/S0002-9297(09)00107-4" target="_blank">
DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources</a>.
<em>Am J Hum Genet</em>. 2009 Apr;84(4):524-33.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19344873" target="_blank">19344873</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2667985/" target="_blank">PMC2667985</a>
</p>
