<h2>Description</h2>
<p>
This track shows multiple alignments of 90 human genomes generated by the Minigraph-Cactus
pangenome pipeline, which creates pangenomes directly from whole-genome alignments. This method
builds graphs containing all forms of genetic variation while allowing use of current mapping and
genotyping tools.
</p>

<h2>Display Conventions and Configuration</h2>
<p>
In full and pack display modes, conservation scores are displayed as a
<em>wiggle track</em> (histogram) in which the height reflects the
size of the score.
The conservation wiggles can be configured in a variety of ways to
highlight different aspects of the displayed information.
Click the <a href="../goldenPath/help/hgWiggleTrackHelp.html"
target="_blank">Graph configuration help</a> link for an explanation
of the configuration options.</p>
<p>
Pairwise alignments of each species to the human genome are
displayed below the conservation histogram as a grayscale density plot (in
pack mode) or as a wiggle (in full mode) that indicates alignment quality.
In dense display mode, conservation is shown in grayscale using
darker values to indicate higher levels of overall conservation
as scored by phastCons. </p>
<p>
Checkboxes on the track configuration page allow selection of the
species to include in the pairwise display.
Note that excluding species from the pairwise display does not alter the
the conservation score display.</p>
<p>
To view detailed information about the alignments at a specific
position, zoom the display in to 30,000 or fewer bases, then click on
the alignment.</p>

<h3>Gap Annotation</h3>
<p>
The <em>Display chains between alignments</em> configuration option
enables display of gaps between alignment blocks in the pairwise alignments in
a manner similar to the Chain track display.  The following
conventions are used:
<ul>
<li><b>Single line:</b> No bases in the aligned species. Possibly due to a
lineage-specific insertion between the aligned blocks in the human genome
or a lineage-specific deletion between the aligned blocks in the aligning
species.
<li><b>Double line:</b> Aligning species has one or more unalignable bases in
the gap region. Possibly due to excessive evolutionary distance between
species or independent indels in the region between the aligned blocks in both
species.
<li><b>Pale yellow coloring:</b> Aligning species has Ns in the gap region.
Reflects uncertainty in the relationship between the DNA of both species, due
to lack of sequence in relevant portions of the aligning species.
</ul></p>

<h3>Genomic Breaks</h3>
<p>
Discontinuities in the genomic context (chromosome, scaffold or region) of the
aligned DNA in the aligning species are shown as follows:
<ul>
<li>
<b>Vertical blue bar:</b> Represents a discontinuity that persists indefinitely
on either side, <em>e.g.</em> a large region of DNA on either side of the bar
comes from a different chromosome in the aligned species due to a large scale
rearrangement.
<li>
<b>Green square brackets:</b> Enclose shorter alignments consisting of DNA from
one genomic context in the aligned species nested inside a larger chain of
alignments from a different genomic context. The alignment within the
brackets may represent a short misalignment, a lineage-specific insertion of a
transposon in the human genome that aligns to a paralogous copy somewhere
else in the aligned species, or other similar occurrence.
</ul></p>

<h3>Base Level</h3>
<p>
When zoomed-in to the base-level display, the track shows the base
composition of each alignment. The numbers and symbols on the Gaps
line indicate the lengths of gaps in the human sequence at those
alignment positions relative to the longest non-human sequence.
If there is sufficient space in the display, the size of the gap is shown.
If the space is insufficient and the gap size is a multiple of 3, a
&quot;*&quot; is displayed; other gap sizes are indicated by &quot;+&quot;.</p>

<h2>Methods</h2>
<p>
The MAF was obtained from the HPRC v1.0 minigraph-cactus HAL file (renamed
to replace all &quot;.&quot; characters in sample names with &quot;#&quot; using
halRenameGenomes) using cactus v2.6.4 as follows.
<pre>
cactus-hal2maf ./js ./hprc-v1.0-mc-grch38.h
al hprc-v1.0-mc-grch38.maf.gz --noAncestors --refGenome GRCh38
--filterGapCausingDupes --chunkSize 100000 --batchCores 96 --batchCount 1
0 --noAncestors --batchParallelTaf 32 --batchSystem slurm --logFile
hprc-v1.0-mc-grch38.maf.gz.log

zcat hprc-v1.0-mc-grch38.maf.gz | mafDuplicateFilter -m - -k | bgzip >
hprc-v1.0-mc-grch38-single-copy.maf.gz</pre>
</p>

<h2>Credits</h2>
<p>
Thank you to Glenn Hickey for providing the HAL file from the HPRC project.
</p>

<h2>References</h2>
<p>
Liao WW, Asri M, Ebler J, Doerr D, Haukness M, Hickey G, Lu S, Lucas JK, Monlong J, Abel HJ <em>et
al</em>.
<a href="https://doi.org/10.1038/s41586-023-05896-x" target="_blank">
A draft human pangenome reference</a>.
<em>Nature</em>. 2023 May;617(7960):312-324.
DOI: <a href="https://doi.org/10.1038/s41586-023-05896-x"
target="_blank">10.1038/s41586-023-05896-x</a>; PMID: <a
href="https://www.ncbi.nlm.nih.gov/pubmed/37165242" target="_blank">37165242</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10172123/" target="_blank">PMC10172123</a>
</p>

<p>
Hickey G, Monlong J, Ebler J, Novak AM, Eizenga JM, Gao Y, Human Pangenome Reference Consortium,
Marschall T, Li H, Paten B.
<a href="https://doi.org/10.1038/s41587-023-01793-w" target="_blank">
Pangenome graph construction from genome alignments with Minigraph-Cactus</a>.
<em>Nat Biotechnol</em>. 2023 May 10;.
DOI: <a href="https://doi.org/10.1038/s41587-023-01793-w"
target="_blank">10.1038/s41587-023-01793-w</a>; PMID: <a
href="https://www.ncbi.nlm.nih.gov/pubmed/37165083" target="_blank">37165083</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10638906/" target="_blank">PMC10638906</a>
</p>

<p>
Armstrong J, Hickey G, Diekhans M, Fiddes IT, Novak AM, Deran A, Fang Q, Xie D, Feng S, Stiller J
<em>et al</em>.
<a href="https://doi.org/10.1038/s41586-020-2871-y" target="_blank">
Progressive Cactus is a multiple-genome aligner for the thousand-genome era</a>.
<em>Nature</em>. 2020 Nov;587(7833):246-251.
DOI: <a href="https://doi.org/10.1038/s41586-020-2871-y"
target="_blank">10.1038/s41586-020-2871-y</a>; PMID: <a
href="https://www.ncbi.nlm.nih.gov/pubmed/33177663" target="_blank">33177663</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7673649/" target="_blank">PMC7673649</a>
</p>

<p>
Paten B, Earl D, Nguyen N, Diekhans M, Zerbino D, Haussler D.
<a href="http://genome.cshlp.org/lookup/pmidlookup?view=long&amp;pmid=21665927" target="_blank">
Cactus: Algorithms for genome multiple sequence alignment</a>.
<em>Genome Res</em>. 2011 Sep;21(9):1512-28.
DOI: <a href="https://doi.org/10.1101/gr.123356.111" target="_blank">10.1101/gr.123356.111</a>;
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21665927" target="_blank">21665927</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3166836/" target="_blank">PMC3166836</a>
</p>
