<H2>Description</H2>
<p>

<div class="warn-note" style="border: 2px solid #9e5900; padding: 5px 20px; background-color: #ffe9cc;">
<p><span style="font-weight: bold; color: #c70000;">NOTE:</span><br>
<b>These data are for research purposes only. While the ClinGen data are 
open to the public, users seeking information about a personal medical or 
genetic condition are urged to consult with a qualified physician for 
diagnosis and for answers to personal medical questions.
</p>
<p> 
UCSC presents these data for use by qualified professionals, and even 
such professionals should use caution in interpreting the significance of 
information found here. No single data point should be taken at face 
value and such data should always be used in conjunction with as much 
corroborating data as possible. No treatment protocols should be 
developed or patient advice given on the basis of these data without 
careful consideration of all possible sources of information.
</p>
<p> 
No attempt to identify individual patients should 
be undertaken. No one is authorized to attempt to identify patients 
by any means.
</p> 
</b>
</div>

<p>
The tracks listed here contain data from the ClinGen ENIGMA BRCA1 and BRCA2 
Expert Panel Specifications to the ACMG/AMP Variant Interpretation 
Guidelines for BRCA1/BRCA1 Version 1.0.0. The ENIGMA VCEP has adapted 
the ACMG-AMP codes for the BRCA1 and BRCA2 genes. These include the 
codes PVS1 (modified PVS1 decision tree), PS3/BS3 (functional data), 
PP4/BP5 (multifactorial data), PM5_PTC (PTC data, at exon level), 
the (potentially) clinically important functional domains defined by 
ENIGMA, and prediction programs (SpliceAI and BayesDel for PP3/BP4).<br><br>
The data required for the application of these ENIGMA codes are displayed in 5 data tracks:</p>
<p>
<ul>
<li><b>BRCA1/BRCA2 protein domains 1.1.0</b> -
Shows the (potentially) clinically important functional domains for 
the genes BRCA1 and BRCA2 as defined by ENIGMA. Data taken from Figure 
1 of the guidelines document CSpec_BRCA12ACMG-Rules-Specifications_V1.0_23-04-27.
</li>
<li><b>BRCA1/BRCA2 exon weights 1.1.0</b> -
This track shows exon-specific weights for the PM5_PTC ACMG code and is used 
for the application of novel protein termination codon (PTC) 
variants in an exon where a different proven pathogenic PTC variant 
has been seen before. Data taken from guidelines document 
CSpec_BRCA12ACMG_Rules-SupplementaryTables_V1.0_23-04-27.
</li>
<li><b>BRCA1/BRCA2 functional assays 1.1.0</b> -
Summary of BRCA1 and BRCA2 functional assay results reviewed for 
application of PS3 and BS3 ACMG codes. Data taken from guidelines 
document CSpec_BRCA12ACMG-Rules-Specifications_V1.0_Table-9_23-04-27.
</li>
<li><b>BRCA1/BRCA2 splicing 1.1.0</b> -
Summary of ACMG codes applicable for variants considered against 
the BRCA1 and BRCA2 PVS1 decision trees. Includes PVS1 and PM5 
codes recommended for initiation, nonsense/frameshift, deletion, 
duplication, and splice site (donor/acceptor ±1,2) variants– 
organized by exon. Data taken from guidelines document 
CSpec_BRCA12ACMG-Rules-Specifications_V1.0_Table-4_23-04-27.
</li>
<li><b>BRCA1/BRCA2 likelihood for PP4 and BP5</b> -
Summary of BRCA1 and BRCA2 multifactorial likelihood analysis 
scores (displayed as Combined LR score) for ACMG codes PP4 and BP5. 
Data taken from Parsons et al. 2019 suppl table HUMU-40-1557-s001_Parson_Multicatorial.xlsx.
</li>
</ul>
</p>
 
<h2>Display Conventions</h2>
<p>
<ul>
<li><b>BRCA1/BRCA2 protein domains 1.1.0</b> -
Items in <b><font color="red">red</font></b> show clinically relevant 
protein domains. Mouseover on items shows the name of the domain and the location.
</li>
<li><b>BRCA1/BRCA2 exon weights 1.1.0</b> -
Mouseover on exons (black items) shows the transcript, exon number, and the PM5_PTC code strength.
</li>
<li><b>BRCA1/BRCA2 functional assays 1.1.0</b> -
Variants assigned with BS3 code are displayed in <b><font color="green">green</font></b>, 
PS3 code in <b><font color="red">red</font></b>, or no code assigned in black. 
Mouseover on items show variant HGVS nomenclature, assigned ACMG code and code weight.
</li>
<li><b>BRCA1/BRCA2 splicing 1.1.0</b> -
Items show variant positions, <b><font color="red">red</font></b> indicates 
exon deletions, <b><font color="blue">blue</font></b> exon duplications, and purple 
items indicate variants with supporting RNA-based functional evidence. Mouseover 
on items show transcript, exon, variant position, possible variant type at 
that position, and assigned ACMG code to the variant.
</li>
<li><b>BRCA1/BRCA2 likelihood for PP4 and BP5</b> -
Variants assigned with PP4 code are displayed in <b><font color="red">red</font></b>, BP5 code 
in <b><font color="green">green</font></b>, and non-informative variants (no code 
assigned) in grey. Mouseover on items show variant HGVS nomenclature, combined 
likelihood ratio (LR) score, and assigned ACMG code and strength.
</li>
</ul>
</p>

<h2>Data Access</h2>
<p>
The most up-to-date VCEP specifications for the application of ACMG/AMP criteria 
for BRCA1 and BRCA2 genes are freely available at the <a target="_blank" 
href="https://cspec.genome.network/cspec/ui/svi/affiliation/50087">ClinGen 
Criteria Specification (CSpec) Registry</a>. This registry is intended to 
provide access to the Criteria Specifications used and applied by <a 
target="_blank" href="https://clinicalgenome.org/affiliation/vcep/#ep_table_heading">ClinGen 
Variant Curation Expert Panels</a> and biocurators in the classification of variants.
</p>

<h2>Methods</h2>
<p>
These data were created and adapted from the files referenced above. Some custom 
scripting was employed in tasks like mapping variants, adding colors and 
mouseovers, and producing the desired format. For the complete details on 
the data processing see the makedoc on our <a target="_blank" 
href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/makeDb/doc/enigma.txt">github</a>.
</p>

<h2>Credits</h2>
<p>
Thank you to Luis Nassar from the Genome Browser team, Anna Benet-Pagès 
and Andreas Laner for technical coordination and consultation, and to 
the ENIGMA consortia for making these data available.</p>

<h2>References</h2>

<p>
Enigma Guidelines: <a target="_blank" href="https://clinicalgenome.org/affiliation/50087/">https://clinicalgenome.org/affiliation/50087/</a></p>
<p>
Enigma Consortium: <a target="_blank" href="https://enigmaconsortium.org/">https://enigmaconsortium.org/</a></p>
<p>
Parsons MT, Tudini E, Li H, Hahnen E, Wappenschmidt B, Feliubadal&#243; L, Aalfs CM, Agata S,
Aittom&#228;ki K, Alducci E <em>et al</em>.
<a href="https://doi.org/10.1002/humu.23818" target="_blank">
Large scale multifactorial likelihood quantitative analysis of BRCA1 and BRCA2 variants: An ENIGMA
resource to support clinical variant classification</a>.
<em>Hum Mutat</em>. 2019 Sep;40(9):1557-1578.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31131967" target="_blank">31131967</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6772163/" target="_blank">PMC6772163</a>
</p>
