<h2>Description</h2>

<p>
This track shows annotations from <a target="_blank"
href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/WebHome">MITOMAP</a>.
MITOMAP is a database of human mitochondrial DNA (mtDNA) information containing
a compilation of mtDNA variation. It allows users to look up human mitochondrial gene 
loci, search for public mitochondrial sequences, and browse or search for reported 
general population nucleotide variants as well as those reported in clinical disease.
</p>
<p>
The data in these tracks are <b>automatically updated from MitoMap weekly</b>.</p>

<h2>Display Conventions and Configuration</h2>
<p>
These data are separated into two tracks:<br><br>
<b>MITOMAP Control and Coding Variants</b><br><br>
This data track contains variants, including mini insertions and deletions, in the
complete mtDNA. The item colors correspond to the variant type:
<b><font color="#53AF01">control region</font></b> vs.
<b><font color="#3E89D3">coding region</font></b>.</p>
<p>
<b>MITOMAP Disease Mutations</b><br><br>
This data track contains disease-annotated mutations (variants) in the
complete mtDNA. The item colors correspond to the variant type:
<b><font color="##015587">coding/control</font></b> vs.
<b><font color="#9933FF">rRNA/tRNA</font></b>.</p>
<p>
For both tracks, item names correspond to the
variant nucleotide change, and mousing over features displays all available
metadata for MITOMAP. Linkouts to the specific MITOMAP datasets are available from
the item description pages, however, you must input the variant on MITOMAP.</p>

<h2>Data Access</h2>
<p>
All source data can be found on the <a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/WebHome">
MITOMAP site</a>.
<p>
The MITOMAP data on the UCSC Genome Browser can be explored interactively with the
<a href="../cgi-bin/hgTables">Table Browser</a> or the
<a href="../cgi-bin/hgIntegrator">Data Integrator</a>.
For automated download and analysis, the genome annotation is stored at UCSC in bigBed
files that can be downloaded from the respective file, e.g. 
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/mitoMapVars.bb" 
target="_blank">MITOMAP Variants</a>, on our download server.
The data may also be explored interactively using our
<a href="../goldenPath/help/api.html" target="_blank">REST API</a>.</p>

<p>
The file for this track may also be locally explored using our tools <tt>bigBedToBed</tt> 
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tools can also be used to obtain features confined to a given range, e.g.,
<br><br>
<tt>bigBedToBed -chrom=chr1 -start=100000 -end=100500 http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/mitoMapVars.bb stdout</tt></p>

<h2>Methods</h2>
<p>
The data were originally downloaded from the <a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/WebHome">
MITOMAP resource</a>. For the <b>Control and Coding Variants track</b>, the following
datasets were combined:
<ul>
<li><a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/VariantsControl">
Control Region Variants (16024-576)</a></li>
<li><a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/VariantsCoding">
Coding & RNA Variants (577-16023, MTTF-MTTP)</a></li></ul>
And for the <b>Disease Mutations track</b>, the following two were combined:
<ul>
<li><a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/MutationsRNA">
rRNA/tRNA Mutations, all</a></li>
<li><a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/MutationsCodingControl">
Coding & Non-Coding/Control Region Mutations, all</a></li></ul></p>
<p>
These tracks have since been updated to automatically fetch files from the MitoMap server.
For all the details on how the data are processed and combined, see
the <a target="_blank"
href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/utils/otto/mitoMap/checkMitoMapUpdate.sh">
MITOMAP makedoc</a>.
</p>

<h2>Credits</h2>
<p>
Thanks to Shiping Zhang and the entire <a target="_blank" href="https://www.mitomap.org/foswiki/bin/view/MITOMAP/WebHome">MITOMAP resource</a>
for making these annotations available.</p>

<h2>References</h2>
<p>
MITOMAP: A Human Mitochondrial Genome Database. http://www.mitomap.org, 2023
</p>
<p>
Lott, M.T., Leipzig, J.N., Derbeneva, O., Xie, H.M., Chalkia, D., Sarmady, M., Procaccio, V., 
and Wallace, D.C. 2013. mtDNA variation and analysis using MITOMAP and MITOMASTER. Current 
Protocols in Bioinformatics 1(123):1.23.1-26. PMID: <a target="_blank" 
href="http://www.ncbi.nlm.nih.gov/pubmed?term=25489354">25489354</a> URL: <a 
target="_blank" href="http://www.mitomap.org/">http://www.mitomap.org</a>
</p>
