<h2>Description</h2>

<p>
The "Splicing Impact" container track contains tracks showing the predicted or validated effect of variants
close to splice sites.
</p>

<h3>SpliceVarDB</h3>
<p>SpliceVarDB is an online database consolidating over 50,000 variants assayed
for their effects on splicing in over 8,000 human genes. The authors evaluated
over 500 published data sources and established a spliceogenicity scale to
standardize, harmonize, and consolidate variant validation data generated by a
range of experimental protocols. Genes and variant locations were obtained using
GENCODE v44. Splice regions were calculated as specific distances from the closest
canonical exon, including 5&apos; and 3&apos; untranslated regions (UTRs). The
database is available at 
<a target=_blank href="https://splicevardb.org">splicevardb.org</a>.</p>

<h2>Display Conventions and Configuration</h2>

<h3>SpliceVarDB</h3>
<p>According to the strength of their supporting
evidence, variants were classified as &quot;splice-altering&quot; (~25%), &quot;not
splice-altering&quot; (~25%), and &quot;low-frequency splice-altering&quot; (~50%), which
correspond to weak or indeterminate evidence of spliceogenicity. 55% of the
splice-altering variants in SpliceVarDB are outside the canonical splice sites
(5.6% are deep intronic). The data is shown as lollipop plots that can be clicked, 
the details page then shows a link to SpliceVarDb with full details.
</p>

<p>The classification thresholds primarily follow those established by the original study.
However, most studies only defined criteria for splice-altering variants and did not define
criteria for variants that resulted in normal splicing. The authors implemented stringent
thresholds to define the normal category and ensure a high-quality set of control variants.
Variants that did not meet these criteria were classified as low-frequency splice-altering
variants with a wide range of sub-optimal scores. Variants that fell between the normal and
splice-altering classifications were placed into a low-frequency splice-altering category.
In situations where a variant was validated multiple times, if at least one validation
returned splice-altering and another returned normal, the &quot;conflicting&quot; category
was applied.
</p>

<P>
The lollipop plots are color-coded based on the <b>score</b> value, which corresponds
to the following classifications:
<ul>
 <li><b>3</b> - <span style="color: rgb(219,61,61);">Splice-altering</span></li>
 <li><b>2</b> - <span style="color: rgb(128,82,160);">Low-frequency</span></li>
 <li><b>1</b> - <span style="color: rgb(57,135,204);">Normal</span></li>
 <li><b>0</b> - <span style="color: rgb(140,140,140);">Conflicting</span></li>
</ul>
</P>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the <a href="../cgi-bin/hgTables">Table Browser</a>
or the <a href="../cgi-bin/hgIntegrator">Data Integrator</a>. The data can be
accessed from scripts through our <a href="https://api.genome.ucsc.edu">API</a>, the track name is
"splicevardb".

<p>
For automated download and analysis, the genome annotation is stored in a bigBed file that
can be downloaded from
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/splicevardb/" target="_blank">our download server</a>.
The file for this track is called <tt>SVADB.bb</tt>. Individual
regions or the whole genome annotation can be obtained using our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool
can also be used to obtain only features within a given range, e.g. 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/hg19/splicevardb/SVADB.bb -chrom=chr21 -start=0 -end=100000000 stdout</tt></p>
</p>

</p>

<h2>Methods</h2>
<p>The data was converted by Patricia Sullivan from SpliceVarDb within a few
hours to bigLolly format and the UCSC Browser staff needed nothing else to do but to downloaded the data.  </p>

<h2>Credits</h2>
<p>Thanks to the SpliceVarDB team for converting the data into our data formats.</p>

<h2>References</h2>
<p>
Sullivan PJ, Quinn JMW, Wu W, Pinese M, Cowley MJ.
<a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(24)00288-X" target="_blank">
    SpliceVarDB: A comprehensive database of experimentally validated human splicing variants</a>.
<em>Am J Hum Genet</em>. 2024 Oct 3;111(10):2164-2175.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39226898" target="_blank">39226898</a>; PMC: <a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11480807/" target="_blank">PMC11480807</a>
</p>
